portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase

```diff
@@ -4524,30 +4881,112 @@ definition {
 	}
 
 	@description = "LPS-86285: Validate in the content page if, the user can remove a section in the experiences that can be different from others experiences (not published)	"
-	@ignore = "true"
+	@ignore = "false"
 	@priority = "3"
 	test RemovingASectionOfDifferentAndUnpublishedExperiences {
+		task ("Go to Experience Content Page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
+		}
 
-		// TODO LPS-86285 RemovingASectionOfDifferentAndUnpublishedExperiences pending implementation
+		task ("Create an experience, add a component and make changes without publish it") {
+			PageEditor.addFragment(
+				collectionName = "Layout Elements",
+				fragmentName = "Container");
 
+			Button.clickPublish();
+
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
+
+			Click(locator1 = "PageEditor#EXPERIENCE_LABEL");
```

Comment:
No need

Reply:
Same as explained here :

If I remove it the test fails with this error:
BUILD FAILED: Element is not present at "//div[contains(@Class,'cke_editable_inline') and contains(@id,'_content')] | //div[contains(@Class,'form-group')]//textarea[contains(@id,'_content')]"
[exec] /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro[editFragmentText]:2171
[exec] /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase[DiscardingChangesFromUnpublishedExperiences]:4159
I think is because when an Experience is added without clicking on Publish, the Selector of Experiences doesn't close so the test can't continue with the next function.

Reply:
You're right